modules/setting/session.go

```diff
@@ -30,6 +34,16 @@ func newSessionService() {
 	SessionConfig.Secure = Cfg.Section("session").Key("COOKIE_SECURE").MustBool(false)
 	SessionConfig.Gclifetime = Cfg.Section("session").Key("GC_INTERVAL_TIME").MustInt64(86400)
 	SessionConfig.Maxlifetime = Cfg.Section("session").Key("SESSION_LIFE_TIME").MustInt64(86400)
+	emptySessionsAreVirtual := Cfg.Section("session").Key("AVOID_CREATING_EMPTY_SESSIONS").MustBool(true)
```

Comment:
Why this is optional? I think it must be true.

Reply:
I guess VirtualSessionProvider changes the semantics of Count. 

I guess I just didn't want to enforce this completely.